src/test/scala/com/microsoft/ml/spark/cognitive/split1/TextAnalyticsSDKSuite.scala

```diff
@@ -147,4 +181,18 @@ class KeyPhraseExtractionSuiteV4 extends TestBase with DataFrameEquality with Te
         row.toSeq.foreach { col => println(col) }
       }
     }
+      test("KPE - Assert Model Version") {
+        val outputCol = extractor.transform(df2)
+          .select("output")
+          .collect()
+        assert(outputCol(0).schema(0).name == "output")
+        val fromRow = KeyPhraseResponseV4.makeFromRowConverter
+        outputCol.foreach(row => {
+          val outResponse = fromRow(row.getAs[GenericRowWithSchema]("output"))
+          assert(outResponse.modelVersion.get == "2021-06-01")
+          })
+        }
+        test("Check Model Version Format"){
+            "2021-06-01".matches("\\d{4}-\\d{2}-\\d{2}")

```

Comment:
Here we should get the model-version and then compare the format, not "2021-06-01"